pkgs/os-specific/darwin/yabai/default.nix

```diff
@@ -52,7 +52,7 @@ in
 
     src = fetchzip {
       url = "https://github.com/koekeishiya/yabai/releases/download/v${version}/yabai-v${version}.tar.gz";
-      sha256 = "sha256-2PH3Hi9x0323MjKHPybNmFddvNNlsaDb1LdiVcZTNJc=";
+      sha256 = "sha256-1szyjcwkhn2wbrcfhh9lh5bnfm1cavxrx6xj4q7521z3zj29a9kf";
```

Comment:
this the stuff behind sha256- isn't a SRI hash

```
❯ nix build ".#yabai"
error: invalid SRI hash '1szyjcwkhn2wbrcfhh9lh5bnfm1cavxrx6xj4q7521z3zj29a9kf'
(use '--show-trace' to show detailed location information)
```

Reply:
https://github.com/NixOS/nixpkgs/pull/242680

Reply:
How did this pass ofborg?

Reply:
ofborg doesn't test anything for yabai it seems?

Reply:
Doesn't ofborg evaluate all changed packages from a PR?

Reply:
Usually, yes, but it seems the way yabai is exposed is a bit different.

Reply:
My bad, apologies for the bad PR. I trusted `nix-prefetch-url`.

Reply:
try `nurl` next time, or `nix-update` directly.

Reply:
Would be nice if something could be done about the blindspot for ofborg.

Reply:
Could this be why `yabai` doesn't show up in search over at [search.nixos.org ](https://search.nixos.org/packages?channel=unstable&type=packages&query=yabai)?

Reply:
Yes